src/main/java/org/galatea/starter/MethodValidationConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.galatea.starter;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.validation.beanvalidation.MethodValidationPostProcessor;
+
+/**
+ * Config for method-level validation (such as validating method input params)
+ */
+@Configuration
+@ComponentScan("org.galatea.starter")
+public class MethodValidationConfig {
+
+  /**
+   * Delegates the Bean Validation to a provider (in our case, the Hibernate Validator).
+   */
+  @Bean
+  public MethodValidationPostProcessor methodValidationPostProcessor() {
```

Comment:
Where do we use this?

Reply:
We don't use it directly.  This is what causes the ConstraintViolationExceptions to be thrown when invalid data is detected (without out, the violations silently slip through the system).